### ISSUE TITLE
"Self-heal Daemon" is not started - GlusterFS 3.8/3.9

### ISSUE THREAD

**Issue by sammcj** [atlantos/nagios-check-gluster#11]
This is a different problem from #10

```bash
root@int-gluster-01:~  #./check_gluster.rb --checks heal
Error: Daemon int-gluster-01.fqdn:"Self-heal Daemon" is not started
```

But there is no daemon for self-heal as far as I'm aware there hasn't been since back in 3.7 which is obviously the current version when this check was written.

I believe this is just part of the main daemon now, all the normal heal commands work on the command line.

**Comment by atlantos** [comment]
@sammcj, thank you for reporting.  I tested this check with Gluster 3.8.5 in #9 and worked. I will try to make testing GlusterFS 3.9 setup and see how it works.

**Comment by sammcj** [comment]
I've fixed it my fork here: https://github.com/sammcj/nagios-check-gluster/blob/master/check_gluster.rb#L174

The script as it stands currently assumes you're using the shortname for the nodes and not their FQDN or their name as specified in DNS, I've made a change to use the name as designated in DNS as that is the most common configuration I've seen, it's also the configuration often used with automation such as puppet modules for GlusterFS.

**Comment by atlantos** [comment]
It's actually not an issue for my environment becauseI use full hostname for linux hostname, so `hostname`, `hostname -f` `getent hosts hostname`, etc are the same.

However I see no objections to change this code. I have updated #12 with my comments.

**Comment by sammcj** [comment]
Interesting, never seen a FQDN used for the shortname before!

**Comment by sammcj** [comment]
#12 Updated, thanks.

**Comment by atlantos** [comment]
Merged #12. @sammcj could you confirm that updated check works on  your GlusterFS 3.8/3.9 setup and this issue can be closed?

**Comment by sammcj** [comment]
Indeed it works on 3.8, I haven't tested it on 3.9 but it should be fine from what I can tell. Thanks for such a quick response!
